articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -74,14 +74,25 @@ The following parameters are used by the App Configuration Push task:
         "TestApp:Settings:Message": "Message data"
     }
     ```
+- **File Content Profile**: The Configuration File's [content profile](https://docs.microsoft.com/azure/azure-app-configuration/concept-config-file). Default value is **Default**.
+     - **Default**: Refers to the conventional configuration file schema widely adopted.
+     - **Kvset**: Refers to a [file schema](https://github.com/Azure/AppConfiguration/blob/main/docs/KVSet/KVSet.v1.0.0.schema.json) that contains all properties of an App Configuration key-value, including key, value, label, content type, and tags.
+- **Import Mode**: The default value is **All**. Determines the behavior when importing key-values.
+    - **All**: Imports all key-values in the configuration file to App Configuration. 
+    - **Ignore-Match**: Imports only settings that have no matching key-value in App Configuration.
+- **Dry Run**: Default value is **Unchecked**.
+   - **Checked**: Prints any updates that would have been performed in a normal run to the console for review.
```

Comment:
We should call out the most important part of this option is that it won't make any real changes to the App Configuration store.

Reply:
Sounds good!